src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -106,43 +106,13 @@ bool ClusterResourceScheduler::NodeAlive(int64_t node_id) const {
 }
 
 bool ClusterResourceScheduler::IsSchedulable(const ResourceRequest &resource_request,
```

Comment:
Do we need this method? Seems like a very thin wrapper

Reply:
Probably could be removed!